account_financial_risk/wizards/parner_risk_exceeded.py

```diff
@@ -31,8 +32,17 @@ def action_show(self):
             "target": "new",
         }
 
+    def _post_account_move_confirm_risk(self, origin_record):
+        template = origin_record.company_id.account_move_confirm_risk_template_id
```

Comment:
Hi, thanks for your suggestions!
Sure this would simplify code for other modules, but I think it would become too hard to understand what's happening as it sometimes happens when reflection is used.

I created this hook to explicitly do things for `account.move` records, (that's why its name is `_post_*account_move*_confirm_risk`); I've added documentation and modified its signature to make it more clear, let me know if that's ok for you.